content/blog/2019-10-22-helm-2150-released.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "Helm 2.15.0 Released"
+authorname: "Matthew Fisher"
+author: "@bacongobbler"
+authorlink: "https://blog.bacongobbler.com"
+date: "2019-10-22"
+---
+
+[Helm 2.15.0](https://github.com/helm/helm/releases/tag/v2.15.0) was released last week. The 2.15.0 release of Helm introduces several improvements to `helm test`. Several commands - `helm search`, `helm repo list`, and `helm install` - received the `--output` flag for machine-readable output.
+
+In addition to these new features (and many more!), many bugs and edge cases in Helm continue to fixed by members of the community. Several parts of the codebase have been refactored for easier maintainability, usability, and better testing.
+
+As Helm moves towards Helm 3's first release, Helm 2 is now transitioning into "maintenance mode". Helm 2.15.0 will be the final feature release for Helm 2.
+
+## Helm 2 support plan
+
+During the transition, we want to take into account any migration issues for users due to the upcoming holiday shopping season and tax season. We also want to clarify what actions may occur after the support contract ends for Helm 2, so that users will not be surprised or caught off guard.
```

Comment:
Can we change the sentence as follows: "During the transition, we want to take into account any migration issues for users due to the upcoming holiday shopping season and tax season." --> "During the transition, we want to take into account any migration issues for users." The sentence seems a bit clunky and unnecessary to mention "upcoming holiday shopping season and tax season". It also may not be applicable or make sense to users in all countries.

Reply:
Good point. I will go ahead and remove that.